samples/linecounter/LineCounter.csproj

```diff
@@ -10,6 +10,7 @@
     <PackageReference Include="Azure.Storage.Blobs" Version="12.6.0" />
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.15.0" />
     <PackageReference Include="Microsoft.Extensions.Azure" Version="1.0.0" />
+    <PackageReference Include="System.Text.Encodings.Web" Version="4.7.2" />
```

Comment:
Should we just rev other azure sdks to a patched version?

Reply:
I think they were already. Our scanner only found these projects with older versions. In this case, the sample doesn't use any of our normal targets so didn't get the update. Not sure why Microsoft.Azure.Core.Spatial wasn't effectively updated, though. The other projects here (they'll likely sign off tomorrow) are track 1.

Reply:
>I think they were already

Not really, the ` Include="Azure.Storage.Blobs" Version="12.6.0" ` is rather old.

>Not sure why Microsoft.Azure.Core.Spatial wasn't effectively updated, though

Hm. It should've been. Maybe the Microsoft.* prefix?

Reply:
Oh, sorry, I misunderstood you. I thought you meant update the reference to STEW in all the other SDKs. In that case, sure, I agree we should probably use latest for all these references while we're at it.